examples/openapi-zod/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "zod-openapi",
+  "scripts": {
+    "dev": "wrangler dev src/index.ts",
+    "deploy": "wrangler deploy --minify src/index.ts",
+    "db:generate": "drizzle-kit generate",
+    "db:migrate": "wrangler d1 migrations apply zod-openapi --local",
+    "db:studio": "drizzle-kit studio",
+    "db:touch": "wrangler d1 execute zod-openapi --local --command='SELECT 1'",
+    "db:migrate:prod": "GOOSIFY_ENV=production drizzle-kit migrate",
+    "db:studio:prod": "GOOSIFY_ENV=production drizzle-kit studio"
```

Comment:
I would like it if we'd had some type checking/linting enabled also for our example projects.